app/src/main/java/chat/rocket/android/fragment/server_config/LoginFragment.kt

```diff
@@ -48,6 +54,38 @@ class LoginFragment : AbstractServerConfigFragment(), LoginContract.View {
         val btnUserRegistration = rootView.findViewById<Button>(R.id.btn_user_registration)
         txtUsername = rootView.findViewById(R.id.editor_username)
         txtPasswd = rootView.findViewById(R.id.editor_passwd)
+        textInputUsername = rootView.findViewById(R.id.text_input_username)
+        textInputPassword = rootView.findViewById(R.id.text_input_passwd)
+
+        //setting text change listeners to username and password edit texts
+        txtUsername.addTextChangedListener(object :TextWatcher
```

Comment:
Thanks @aniketsingh03 for this PR!

In the future I'd go to recommend you the RxBinding. Your code isn't wrong, but you can avoid the boilerplate. 

Just a friendly recommendation.

Reply:
Yeah I'll implement it. Thanks for the advice @filipedelimabrito !